src/main/java/org/opensearch/neuralsearch/search/collector/HybridCollapsingTopDocsCollector.java

```diff
@@ -72,30 +72,26 @@ public class HybridCollapsingTopDocsCollector<T> implements HybridSearchCollecto
         this.groupSelector = groupSelector;
         this.collapseField = collapseField;
         this.sort = groupSort;
-        if (topNGroups < 1) {
```

Comment:
we have dropped this validation, is this intentional, what's the reason?

Reply:
The whole purpose of this PR is to remove this condition because when numHits becomes 0, then topNGroups also becomes 0 and throws the error which we are facing while running integ test. By removing this condition, the normal workflow will run and it will handle the case when results are not present.

Reply:
Does it mean `topNGroups must be >= 1 (got " + topNGroups + ")` is not correct? topNGroup can be zero or below zero?

Reply:
Yes @heemin32 . It is not correct.